### DISCUSSION TITLE
Run Java Project of spark agent in IntelliJ

### DISCUSSION THREAD

**Discussion by zacayd** [AbsaOSS/spline-spark-agent#533] [Q&A]
HI
what steps i need to run the java project of Spark Agent in  my IntelliJ
I get an error of
```
C:\Spline\core\src\main\scala\za\co\absa\spline\harvester\LineageHarvester.scala:25:8
object DefaultSplineConfigurer is not a member of package za.co.absa.spline.harvester.conf
import za.co.absa.spline.harvester.conf.DefaultSplineConfigurer
```

**Comment by wajda** [comment]
What do you mean by "run project"? What exactly are you trying to do? What commands do you execute?

**Comment by zacayd** [comment]
Hi Alex- i ment- Built Project on InteliJ

**Reply by wajda** [reply]
It seems your project structure is somehow broken. Try to re-create a project from the POM file, and it should just work.

**Reply by cerveada** [reply]
After creating the project from maven pom files. I usually build the project using the maven tab in IntelliJ.
